app/controllers/admin/users_controller.rb

```diff
@@ -49,6 +49,14 @@ def user_status
       redirect_to "/admin/users/#{@user.id}/edit"
     end
 
+    def export_data
+      user = User.find(params[:id])
+      send_to_admin = JSON.parse(params[:send_to_admin])
+      ExportContentWorker.perform_async(user.id, send_to_admin: send_to_admin)
+      flash[:success] = "Data exported to the #{send_to_admin ? 'admin' : 'user'}. The job will complete momentarily."
```

Comment:
Rather than propagating this boolean variable all the way through this logic flow, how about we use it here to set the email we want to send to and then we send that email to the worker. I think 
```ruby
ExportContentWorker.perform_async(user.id, email_address)
```
is a bit cleaner and more reusable for other scenarios.

Reply:
Agree, it looks like we only use the email of the recipient down in the emailer and the email templates so, we should probably just probably just send the ID of the user to export and the email address of the recipient.
It'd simplify a lot of code actually :)